views/default/admin_domains_pricing.pdt

```diff
@@ -412,9 +412,15 @@
         });
 
         // Set the module email template as the email content
-        $('.load_sample_email').on('click', function() {
-            var templates = <?php echo json_encode($package_fields['template'], JSON_PRETTY_PRINT);?>;
+        var templates = <?php echo json_encode($package_fields['template'], JSON_PRETTY_PRINT);?>;
+
+        if (typeof templates == 'undefined' || templates.length == 0) {
```

Comment:
You'll need to do this
`        if (typeof templates == 'undefined'  || templates === null || templates.length == 0) {`
in case the json is invalid.  This occurred for me with the Enom module.

Reply:
Updated